portfolio/src/main/java/com/google/sps/servlets/HistoryServlet.java

```diff
@@ -78,7 +84,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         score = (double) entity.getProperty("score");
       }
 
-      Comment comment = new Comment(id, name, email, message, timestamp, score);
+      boolean usercomment;
+      //Check if comment is written by current user
+      if (email.equals(userService.getCurrentUser().getEmail())) {
+        usercomment = true;
```

Comment:
nit: variable naming in lowerCamelCase: https://google.github.io/styleguide/javaguide.html#s5.2.6-parameter-names

also consider renaming it to be more readable like isAuthor

you could also use one line:

boolean isAuthor = email.equals(userService.getCurrentUser().getEmail());